### DISCUSSION TITLE
2 profiles on Windows Terminal

### DISCUSSION THREAD

**Discussion by Celibioux** [microsoft/WSL#9440] [Q&A]
Hello Microsoft
After Installing WSL 2 on a New computer ive got two profile on the terminal one called ubuntu and the other linux is it normal or i did  something wrong ?

**Comment by Biswa96** [comment]
Windows Terminal automatically creates profiles for each distribution installed in WSL. You can find list of installed distributions in WSL with `wsl.exe -l -v` command.

**Comment by Celibioux** [comment]
I know that Windows Terminal Scans the system for all installed distros but i have installed only ubuntu via the wsl --install and on Windows Terminal i have two profiles one is the Ubuntu and the Other is Linux (Terminal.Wsl ) i think because i don't have my laptop now to check the exact profile naming

**Comment by NotTheDr01ds** [comment]
You may have been running into something like the following Super User questions:

* [Different profiles of the same WSL2 linux instance in Windows Terminal](https://superuser.com/q/1737942/1210833)
* [Can't delete or hide duplicate Ubuntu Profile in Windows Terminal](https://superuser.com/q/1737991/1210833)
